cmdeploy/pyproject.toml

```diff
@@ -18,6 +18,7 @@ dependencies = [
   "ruff",
   "pytest",
   "pytest-xdist", 
+  "execnet",
```

Comment:
it's already a dependency via pytest-xdist but better to spell it out explicitely.